exercises/armstrong-numbers/description.md

```diff
@@ -0,0 +1,10 @@
+An [Armstrong number](https://en.wikipedia.org/wiki/Narcissistic_number) is a number that is the sum of its own digits each raised to the power of the number of digits.
+It also depends on the base of the number being used (for example base = 10 for the decimal system or base = 2 for the binary system).
+
+For example:<sup></sup>
+153<sub>10</sub> is &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;an armstrong number in base 10, because: 153<sub>10</sub> = 1<sup>3</sup> + 5<sup>3</sup> + 3<sup>3</sup> = 1 + 125 + 27 = 154<sub>10</sub>
```

Comment:
Although I agree that using `<sup></sup>` makes for the best display of exponentiation, I don't really like using HTML in markdown. I've looked at the all-your-base exercise, and [it's description](https://github.com/exercism/problem-specifications/blob/master/exercises/all-your-base/description.md) uses the caret sign (^), which I think is still quite readable.

Reply:
To be consistent here, I'd also remove the "sub" additional to the "sup" then.
This would "cost" quite a bit of readability for the rendered version because we lose the hint in what base a number is represented. If we decide to drop all html, then I'd also vote for dropping the "base" completely and go with @Insti argument at the bottom. Do we have a consensus here?